### ISSUE TITLE
self-closing rule not working

### ISSUE THREAD

**Issue by zigomir** [meteorlxy/eslint-plugin-prettier-vue#16]
eslint will ignore [self–closing SFC component rule](https://eslint.vuejs.org/rules/html-self-closing.html) when I have `"prettier/vue"` listed in my eslint config like so

```json
"extends": [
  "standard",
  "plugin:@typescript-eslint/recommended",
  "plugin:vue/vue3-recommended",
  "plugin:prettier-vue/recommended",
  "prettier/standard",
  "prettier/vue"
]
```

Not sure where exactly the issues lies, but I though I should report it here.

I'm using Vue 3 and `eslint-plugin-prettier-vue: 3.0.0-alpha.2`.

**Comment by meteorlxy** [comment]
You must understand prettier & eslint first.

If you are using prettier, then almost all of the "code styles" are enforced to be "prettier style". That mean, all eslint "code style rules" should be disabled, and everything related to code style is handled by prettier only.

That's the reason why prettier calls itself an **opinionated** code formatter

**Comment by zigomir** [comment]
👍 got it, thanks!
